src/bp/core/app.inversify.ts

```diff
@@ -134,6 +136,16 @@ container
   .to(LegacyStats)
   .inSingletonScope()
 
+container
+  .bind<RolesStats>(TYPES.RolesStats)
+  .to(RolesStats)
+  .inSingletonScope()
+
+container
+  .bind<SDKStats>(TYPES.SDKStats)
+  .to(SDKStats)
+  .inSingletonScope()
+
```

Comment:
The file `app.inversify` contains only the core containers of the app. Since we're adding multiple stats containers, please move them to `telemetry.inversify.ts`. For the file structure, check `services.inversify.ts`

Reply:
fixed: b9ddf2f2408890d0073541ea35ec73aa23196265